content/pandas/concepts/dataframe/terms/groupby/groupby.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: '.groupby()'
+Description: 'Groups a DataFrame using a mapper or a series of columns and returns a GroupBy object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Structures'
+  - 'Pandas'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The `.groupby()` function groups a [DataFrame](https://www.codecademy.com/resources/docs/pandas/dataframe) using a mapper or a series of columns and returns a [GroupBy](https://www.codecademy.com/resources/docs/pandas/groupby) object. A range of methods, as well as custom functions, can be applied to GroupBy objects in order to combine or transform large amounts of data in these groups.
+
+## Syntax
+
+```pseudo
+dataframevalue.groupby(by, axis, level, as_index, sort, group_keys, observed, dropna)
+```
+
+`.groupby()` uses the following parameters:
+
+- `by`: if a dict or Series is passed, the values will determine groups. If a list or [ndarray](https://www.codecademy.com/resources/docs/numpy/ndarray) with the same length as the selected axis, the values will be used to form groups. A label or list of labels can be used to group by a particular column or columns.
+- `axis`: Split along rows (0 or "index") or columns (1 or "columns"). Default value is 0.
+- `level`: If the axis is a MultiIndex, group by a particular level or levels. Value is int or level name, or sequence of them. Default value is `None`.
+- `as_index`: Boolean value. `True` returns group labels as index in aggregated output, `False` returns labels as DataFrame columns. Default value is `True`.
+- `sort`: Boolean value. `True` sorts the group keys. Default value is `True`.
+- `group_keys`: Boolean value. Add group keys to index when calling apply. Default value is `True`.
+- `observed`: Boolean value: If `True` only show observed values for categorical groupers, otherwise show all values. Default value is `False`.
```

Comment:
```suggestion
- `observed`: Boolean value. If `True`, only show observed values for categorical groupers, otherwise show all values. Default value is `False`.
```